literature.bib

```diff
@@ -1,3 +1,17 @@
+
+@inproceedings{PAM+:ICPE20,
+  title={Sampling Effect on Performance Prediction of Configurable Systems: A Case Study},
+  author={Juliana Alves Pereira and Mathieu Acher and Hugo Martib and Jean-Marc J{\'e}z{\'e}quel},
+  booktitle={Proceedings of the ACM/SPEC International Conference on Performance Engineering},
```

Comment:
Only use abbreviations for conferences as defined in MyAbrv

Reply:
add something like

@String{ICPE = "Proc.\ Int'l Conf.\ on Performance Engineering (ICPE)"}

to https://github.com/tthuem/BibTags/blob/master/MYabrv.bib

Reply:
Check